docs/overview.rst

```diff
@@ -0,0 +1,45 @@
+Overview
+========
+
+
+Start your own Vyper project
+----------------------------
+
+Twig lets you write tests using ``web3.py`` and ``pytest-ethereum`` against your ``Vyper`` smart contracts.
+
+
+Twig setup
+~~~~~~~~~~
+
+- clone twig repo
+- ``python -m venv venv`` (note: requires python3.6)
+- ``. venv/bin/activate``
+- ``pip install -e .``
+
+
+Project setup
+~~~~~~~~~~~~~
+
+.. NOTE:: ``twig/contracts/name_registry.vy`` and ``tests/contracts/test_name_registry.py`` have been included as examples on how to get started, but are safe to delete.
+
+- create your vyper contract file in ``twig/contracts/``
+  (i.e. ``twig/contracts/example.vy``)
+
+- write your vyper contract test file in ``tests/contracts/``
```

Comment:
Is the extra directory name `contracts` required in any way or is it just convention?  If convention, it might be good to remove it in order to simplify the directory structure for the example.

Reply:
there are other test files in `root/tests/` (i.e. for core twig functionality), and I was thinking `root/tests/contracts/` would be where the user would write their contract tests and help separate from core functionality tests - would it be better to have a separate tests folder in the root for user's contract tests? i.e. `root/project_tests/` or move core tests to `root/core_tests/`?

Reply:
I can see choosing that convention for a project, but for an example I think it's good to keep it as simple as possible which for me suggests the following structure:

```
--- root/
 |--- contracts/
 |  |---registry.vy
 |
 |--- tests/
    |---test_registry.py
```

The inclusion of an extra directory layer could easily be confused as a requirements for `twig` to work as opposed what it actually is, just a convention for keeping things organized.